src/Tests/Service/U2fServiceAuthenticationVerificationTest.php

```diff
@@ -229,13 +240,18 @@ public function it_throws_unexpected_u2f_registration_verification_errors($error
         $response->keyHandle = $keyHandle;
         $response->signatureData = 'signature-data';
 
+        $yubicoResponse = new \stdClass;
+        $yubicoResponse->keyHandle     = $response->keyHandle;
+        $yubicoResponse->signatureData = $response->signatureData;
+        $yubicoResponse->clientData    = $response->clientData;
+
         $u2f = m::mock('u2flib_server\U2F');
         $u2f->shouldReceive('doAuthenticate')
             ->once()
-            ->with(m::anyOf([$yubicoRequest]), m::anyOf([$yubicoRegistration]), m::anyOf($response))
+            ->with(m::anyOf([$yubicoRequest]), m::anyOf([$yubicoRegistration]), m::anyOf($yubicoResponse))
             ->andThrow(new Error('error', $errorCode));
 
-        $service = new AuthenticationVerificationService($u2f);
+        $service = new U2fService(new AppId(self::APP_ID), $u2f);
 
         $this->setExpectedExceptionRegExp('Surfnet\StepupU2fBundle\Exception\LogicException');
```

Comment:
why use `setExpectedExceptionRegExp()` when no RegExp is used? Please use `setExpectedException()` or add a RegExp to test :wink: (the same in the Registration test please :smile: )